packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/getMouseEventInfo.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @internal
+ * Get pageX of a MouseEvent.
+ * For android WebViews e.pageX will be undefined, get info from TouchEvent instead
+ * @param e
+ * @returns pageX number
+ */
+export function getPageX(e: MouseEvent): number {
+    let pageX = e.pageX;
+    if (!pageX) {
+        const touchEvent = e as unknown as TouchEvent;
+        const touch = touchEvent.targetTouches[0];
+        pageX = touch.pageX;
+    }
+    return pageX;
+}
+
+/**
+ * @internal
+ * Get pageY of a MouseEvent.
+ * For android WebViews e.pageY will be undefined, get info from TouchEvent instead
+ * @param e
+ * @returns pageY number
+ */
+export function getPageY(e: MouseEvent): number {
+    let pageY = e.pageY;
+    if (!pageY) {
+        const touchEvent = e as unknown as TouchEvent;
+        const touch = touchEvent.targetTouches[0];
+        pageY = touch.pageX;
+    }
+    return pageY;
+}
```

Comment:
These new functions are only used in DragAndDropHelper, do you think we can move them into DragAndDropHelper file? 
Also, I think if you would prefer to have them in a different file it would be better to have them in plugin utils instead of having them in packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/getMouseEventInfo.ts

Reply:
Updated! Thanks!